<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioDialog</class>
 <widget class="QDialog" name="AudioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>460</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Audio Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabOutput">
      <attribute name="title">
       <string>Audio Output</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeWidget" name="audioOutTree">
         <property name="indentation">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <column>
          <property name="text">
           <string notr="true">SD</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Device</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="sampleRateLayout">
         <item>
          <widget class="QLabel" name="outputSampleRateLabel">
           <property name="text">
            <string>Rate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="outputSampleRate">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Audio output sample rate</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>8000</number>
           </property>
           <property name="maximum">
            <number>192000</number>
           </property>
           <property name="singleStep">
            <number>1000</number>
           </property>
           <property name="value">
            <number>48000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="outputReset">
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Reset values to defaults</string>
           </property>
           <property name="text">
            <string>R</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="outputFormatLayout">
         <property name="topMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QComboBox" name="outputUDPChannelMode">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Channel copy mode</string>
           </property>
           <item>
            <property name="text">
             <string>Left</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Right</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mixed</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Stereo</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="decimationLabel">
           <property name="text">
            <string>/</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="decimationFactor">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Decimation</string>
           </property>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="outputUDPChannelCodec">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>65</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Encoding</string>
           </property>
           <item>
            <property name="text">
             <string>L16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>L8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PCMA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PCMU</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>G722</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Opus</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="outputSDPText">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Encoding string in SDP format</string>
           </property>
           <property name="text">
            <string>PCMA/48000</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="outputUDPLayout">
         <item>
          <widget class="QLabel" name="outputUDPAddressLabel">
           <property name="text">
            <string>Addr</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="outputUDPAddress">
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Destination UDP address</string>
           </property>
           <property name="inputMask">
            <string>000.000.000.000</string>
           </property>
           <property name="text">
            <string>127.0.0.1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="outputUDPPortLabel">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="outputUDPPort">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Destination UDP port</string>
           </property>
           <property name="inputMask">
            <string>00000</string>
           </property>
           <property name="text">
            <string>9998</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="ButtonSwitch" name="outputUDPCopy">
           <property name="toolTip">
            <string>Copy audio to UDP</string>
           </property>
           <property name="text">
            <string>UDP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="outputUDPUseRTP">
           <property name="toolTip">
            <string>Use RTP protocol</string>
           </property>
           <property name="text">
            <string>RTP</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="outputGeneralLayout">
         <item>
          <widget class="QPushButton" name="outputCleanup">
           <property name="toolTip">
            <string>Remove references to devices not listed</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/recycle.png</normaloff>:/recycle.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ButtonSwitch" name="outputResetKey">
           <property name="toolTip">
            <string>Remove settings for this device and return to defaults</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/bin.png</normaloff>:/bin.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabInput">
      <attribute name="title">
       <string>Audio Input</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeWidget" name="audioInTree">
         <property name="indentation">
          <number>0</number>
         </property>
         <column>
          <property name="text">
           <string>SD</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Device</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="inputVolumeLayout">
         <item>
          <widget class="QLabel" name="inputSampleRateLabel">
           <property name="text">
            <string>Rate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="inputSampleRate">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Audio input sample rate</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>8000</number>
           </property>
           <property name="maximum">
            <number>192000</number>
           </property>
           <property name="singleStep">
            <number>1000</number>
           </property>
           <property name="value">
            <number>48000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="inputVolumeLabel">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Volume</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="inputVolume">
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Input volume</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="inputVolumeText">
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>0.50</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="inputReset">
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Reset values to default</string>
           </property>
           <property name="text">
            <string>R</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="inputGeneralLayout">
         <item>
          <widget class="QPushButton" name="inputCleanup">
           <property name="toolTip">
            <string>Remove references to devices not listed</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/recycle.png</normaloff>:/recycle.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ButtonSwitch" name="inputResetKey">
           <property name="toolTip">
            <string>Remove settings for this device and return to defaults</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/bin.png</normaloff>:/bin.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>audioOutTree</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AudioDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AudioDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
